doc/compose.md

```diff
@@ -33,10 +33,16 @@ This is a beta feature, please [join our community](https://ploomber.io/communit
 let us know how we can improve it!
 ```
 
-JupySQL allows you to break queries into multiple cells, simplifying the process of
-building large queries.
+Tired of wrangling complex, large-scale SQL queries? JupySQL is here to revolutionize your experience with an intuitive, user-friendly package specifically designed to help you manage and organize large SQL queries in Jupyter Notebooks.
 
-As an example, we are using a sales database from a record store. We'll find the
+- **Simplify your workflow:** JupySQL allows you to break down large SQL queries into manageable chunks and organize them within your Jupyter Notebook, enhancing readability and maintainability.
```

Comment:
I think this list is good, but let's make it shorter

Reply:
Resolved in https://github.com/ploomber/jupysql/pull/300/commits/50219a0c10c5aa8bc64bbbd849e4e41f945ade1d